ui/src/childinfo/ChildInfo.test.js

```diff
@@ -8,8 +8,9 @@ describe('Initial Render', () => {
   const testName = 'John'
   const age = 5;
   let childInfo;
+  let childId = 'some id string'
   beforeEach(() => {
-    childInfo = shallow(<ChildInfo name={testName} age={age}/>);
+    childInfo = shallow(<ChildInfo name={testName} age={age} childId={childId} />);
```

Comment:
Don't think we will have childId at this point, that will be received after a post to makeAWish

Reply:
This still needs to be removed